src/content/docs/alerts/create-alert/condition-details/title-template.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Title template for NRQL alert conditions
+tags:
+  - Alerts and applied intelligence
+  - Alerts
+  - Alert conditions
+metaDescription: 'Use the title template when you create an alert condition so you can easily identify related incidents.'
+redirects:
+  - /docs/alerts/new-relic-alerts/defining-conditions/alert-condition-descriptions
+  - /docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-descriptions
+  - /docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-custom-violation-descriptions
+  - /docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/alert-custom-violation-descriptions/
+freshnessValidatedDate: never
+---
+
+import alertsAddingAlertDetailstoanAlertCondition from 'images/alerts_screenshot-crop_adding-alert-details-to-an-alert-condition.webp'
+
+
+A title template is used when incidents are opened by the condition. It overrides the default title.
+
+## Why use a title template? [#why-titles]
+
+An alert condition is the foundation for incident creation. Whether you're using our guided mode or crafting a condition from a chart or policy, using a unique title template is the final step to creating an alert condition. While this is optional, we recommend it. An alert condition defines a set of thresholds you want to monitor. If any of those thresholds are breached, an incident is created. Meaningful title templates help you pinpoint issues and resolve outages faster.
```

Comment:
To break up this content, all of which is extremely helpful + good to know, I might add bullet points here, maybe something like... 

While this is optional, we recommend it:

* An alert condition defines...
* If any of those thresholds are breached...
*Meaningful title....

Reply:
actually you use bullets a bit in this doc. maybe just a paragraph break? i think this is just for a skimmability fix :)

Reply:
I edited it to make it easier to skim! good callout!